restapi/src/main/java/io/stargate/web/impl/RestApiServer.java

```diff
@@ -156,6 +163,7 @@ protected void configure() {
     environment.jersey().register(CollectionsResource.class);
     environment.jersey().register(NamespacesResource.class);
 
+    // Swagger endpoints
     environment.jersey().register(ApiListingResource.class);
     environment.jersey().register(SwaggerSerializers.class);
```

Comment:
This could very likely be a singleton too, if it matters.

Reply:
Yep, I will do this on top. Can you move Swagger resource to this group if you are grouping already?